src/main/java/com/github/sh0ckr6/achievementborder/commands/ConfigCommand.java

```diff
@@ -202,4 +203,30 @@ private void handleEdit(CommandSender sender, String[] args) {
                        + ChatColor.GREEN + "! New value: "
                        + ChatColor.GOLD + value);
   }
+
+  /**
+   * Handle <code>reset</code> subcommand
+   *
+   * @param sender The {@link CommandSender} that sent the command
+   * @param args The arguments of the subcommand
+   * @author sh0ckR6
+   * @since latest
+   */
+  private void handleReset(CommandSender sender, String[] args) {
+    String configName = args[0];
```

Comment:
```suggestion
    if (args.length == 0) {
      sender.sendMessage(ChatColor.RED + "Please provide a configuration name");
      return;
    }
    String configName = args[0];
```
Arguments can be empty causing `String configName = args[0];` to throw an `ArrayIndexOutOfBoundsException`